src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/AndroidResourceTests.cs

```diff
@@ -66,7 +68,7 @@ public void MenuActionLayout ()
 		app:showAsAction = ""never"" />
 
 </menu>");
-			Monodroid.AndroidResource.UpdateXmlResource (actions, new Dictionary<string, string> (), null);
+			Monodroid.AndroidResource.UpdateXmlResource (actions, xmlFilePath, new Dictionary<string, string> (), null);
```

Comment:
I think this should be `Path.Combine (path, "res")`

Reply:
You mean instead of `actions`?

Reply:
Instead of xmlFilePath. The ‘res’ Param should be te root of the resource directory. I know why this happened now. There were two PR's one altered the `UpdateXmlResource` signature, bother were open at the the time the commits were made so they both passed. One was merged, the other which added the test was not rebased before merging, so it looked like it was working.